Cargo.toml

```diff
@@ -35,8 +35,8 @@ futures-executor = "0.3.21"
 futures-util = { version = "0.3.26", default-features = false, features = ["alloc"] }
 http = "0.2.6"
 itertools = "0.12.0"
-ruma = { version = "0.9.3", features = ["client-api-c", "compat-upload-signatures", "compat-user-id", "compat-arbitrary-length-ids", "unstable-msc3401"] }
-ruma-common = "0.12.0"
+ruma = { git = "https://github.com/ruma/ruma", rev = "df1a63909ab7cc815ff8d6e292965c82b82d697e", features = ["client-api-c", "compat-upload-signatures", "compat-user-id", "compat-arbitrary-length-ids", "unstable-msc3401"] }
```

Comment:
Why do we need the custom Ruma branch? Could we get a small patch release for ruma please? :pray:

Reply:
There's a breaking change in `PushConditionRoomCtx` for this, so it would require a major version bump.

Reply:
It'd be really great if we didn't have to use a custom branch; in addition to adding to download times for SDK developers, it's also a blocker for a (awaited for so long) new release of the SDK.

Reply:
We don't want to release a major version now and risk having to do another one before the release of the SDK because of another potential fix. It should be easy for us to make the release when the SDK needs it. 

Is there a date for the SDK release yet?

Reply:
Nah, we don't really have a release date. Ok, it's unfortunate but acceptable for the time being.